libs-scala/flyway-testing/hash-migrations.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+set -u
```

Comment:
```suggestion
set -eou pipefail
```
just because we use this in most other places and it removes the potential confusion on whether the difference is intentional/required.

Reply:
done in all three scripts